src/Entities/Boat.h

```diff
@@ -78,6 +79,7 @@ class cBoat :
 
 	void UpdatePaddles(bool rightPaddleUsed, bool leftPaddleUsed);
 private:
+	Vector3d m_LastSentPosition;
```

Comment:
You should make this `protected` in `cEntity`, otherwise all the logic in `cEntity` will be using the wrong coords.